src/main.rs

```diff
@@ -55,6 +55,10 @@ struct Cli {
 	/// Disable searching through Home Manager packages
 	#[clap(long = "no-home-manager-packages", action = ArgAction::SetFalse)]
 	home_manager_packages: bool,
+
+	/// Filter out pull requests that are not updates
+	#[clap(long)]
+	updates_only: bool,
```

Comment:
```suggestion
	updates_only: bool,
```
I kinda prefer `--only-updates` over `--updates-only` tbh. Sorry for the nitpick lol.